README.md

```diff
@@ -170,6 +171,25 @@ https://demo.imgix.net/image.jpg?w=100&dpr=4&q=23 4x,
 https://demo.imgix.net/image.jpg?w=100&dpr=5&q=20 5x
 ```
 
+#### Variable Qualities
+
+By default, this library will automatically append a variable `q` parameter mapped to each `dpr` parameter when generating a [fixed-width image](#fixed-width-images) srcset.
+
+To customize variable qualities, you can pass a `variable_qualities` dictionary in the `options` while creating srcset as below:
+```py
+>>> from imgix import UrlBuilder
+>>> client = UrlBuilder("sdk-test.imgix.net")
```

Comment:
Let's keep the image domain/path consistent with what's already demo'd in the documentation, e.g. `https://demo.imgix.net/image.jpg?...`

We'll need to change this in the [outputted results](https://github.com/imgix/imgix-python/pull/102/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R186-R190) section as well.

```suggestion
builder = UrlBuilder("demo.imgix.net")
```